### PR TITLE
AiSD-7-bubbleSort

### PR DESCRIPTION
**Pull request by mztezi**

### CODE REVIEW

File: AiSD-7-bubbleSort.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+function swapNeat(arr, index1, index2) {
+    [arr[index1], arr[index2]] = [arr[index2], arr[index1]];
+}
+
+function swap(arr, index1, index2) {
+    let tempVal = arr[index1];
```
Comment (kdryzek) [inline AiSD-7-bubbleSort.js]: const

File: bubbleSort.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                swapped = true;
+            }
+        }
+    } while (swapped)
+}
+
+function bubbleSortAlt(arr) {
+    let swapped;
+
+    for( ; ; k++) {
```
Comment (kdryzek) [inline bubbleSort.js]: xd; do not use swapped boolean

File: AiSD-7-bubbleSort.js (hunk 1 above)
Comment (mztezi) [reply AiSD-7-bubbleSort.js]: fixed

File: bubbleSort.js (hunk 2 above)
Comment (mztezi) [reply bubbleSort.js]: changed - fixed

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (kdryzek) [inline .vscode/settings.json:1]: add .vscode/ to gitignore

Comment (mztezi) [reply .vscode/settings.json:1]: added

**Review by kdryzek — APPROVED** [review]

File: src/swap.js
Comment (kdryzek) [inline src/swap.js:1]: put it into src/common/swap.js or src/utils/swap.js

Comment (mztezi) [reply src/swap.js:1]: moved

**Pull request merged**